### DISCUSSION TITLE
Could not copy ... Error Operation not permitted. Retrying in five minutes.

### DISCUSSION THREAD

**Discussion by 125devin** [Chia-Network/chia-blockchain#8040] [Support]
While plotting with Ubuntu, plotter log files have the message, "Could not copy ... Error Operation not permitted. Retrying in five minutes." The chia version is 1.2.3.  The plot file is created at the destination but its size is zero. 

The mount point for the destination drive is on a root level directory with permissions "drwxrwx---" and owner "root plugdev". The user id that the chia plotting process is running under is a member of the group plugdev. The destination drive is an external USB drive formatted with NTFS. 

The plot destination directory was recently renamed. 

What will fix this?

**Comment by 125devin** [comment]
Here's what I did:

Removed the plot directory from the plots screen in the Chia GUI.
Unmounted the drive.
Commented out the lines in /etc/fstab that mount the drive at boot. (That is intended only for permanently installed internal drives.)
Unplugged and plugged the USB cable to the drive.
Changed the old mount point for the drive to a symbolic link to the new mount point automatically created under the directory /media .
Wondered some more why it stopped working the other way.

The new mount point and plot directory are no longer owned by user root, but the login user instead. The next scheduled plot job copied just fine when the plot was finished.

**Comment by BrandtH22** [comment]
@125devin , based on your previous message it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
